.github/workflows/_integration_tests.yaml

```diff
@@ -72,22 +72,23 @@ jobs:
             test: dbless
           - name: postgres
             test: postgres
+            router-flavor: 'traditional_compatible'
```

Comment:
Do we have an issue tracking the support of expressions in DB backed mode? (I don't recall if the plans were to support it at some point or not 🤔 ) I'd explicitly at this here for future readers so that they know why there's difference.

Reply:
It doesn't look like we have any good public information about it on brief review. I don't see anything in the proxy issue tracker or in https://docs.konghq.com/gateway/3.4.x/key-concepts/routes/expressions/ or the 3.0 proxy breaking changes list.

Reply:
I'd suggest we create one and reference here as we might a) loose track what's default/supported where and b) if we're testing the non default router-flavors